app/models/SalesFromEuWithOptionalVat.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{Json, OFormat}
+
+// TODO: This will eventually be fleshed out to represent all the data of a sale from NI
```

Comment:
Is this TODO outstanding?

Reply:
It is in a sense - this class is a duplicate of SalesFromEu, but with salesFromCountry changed to Optional. The TODO was added by "Wilson" in August - should we remove both?

Reply:
I think we can remove both then